### PR TITLE
Allow disabling batch upload.

### LINKED ISSUE #493: Allow disabling batch upload.

### PR DESCRIPTION
**Pull request by mjgiarlo**
Fixes #493

### CODE REVIEW

File: app/views/hyrax/base/_form.html.erb [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <% unless f.object.persisted? %>
     <% provide :metadata_tab do %>
       <p class="switch-upload-type">To create a separate work for each of the files, go to <%= link_to "Batch upload", hyrax.new_batch_upload_path %></p>
-    <% end %>
+    <% end if Flipflop.enable_batch_upload? %>
```
Comment (jcoyne) [inline app/views/hyrax/base/_form.html.erb]: can we just combine this condition with the one on line 14?
```ruby
if Flipflop.enable_batch_upload? && f.object.new_record?
```

Comment (mjgiarlo) [inline app/views/hyrax/base/_form.html.erb]: Good :eye:!

File: app/controllers/concerns/hyrax/batch_uploads_controller_behavior.rb [hunk 2]
```diff
@@ -42,6 +43,19 @@ def build_form
         @form.payload_concern = params[:payload_concern]
       end
 
+      def redirect_when_feature_disabled!
+        return if Flipflop.enable_batch_upload?
+        respond_to do |wants|
```
Comment (mjgiarlo) [inline app/controllers/concerns/hyrax/batch_uploads_controller_behavior.rb]: @jcoyne I followed up on your suggestion and I'm seeing two spec failures w/ the batch_uploads_controller_spec below. I've tried a few different incantations, but the `wants` object here never seems to trigger the `json` or `html` blocks below:

```
(byebug) p wants
#<ActionController::MimeResponds::Collector:0x000000081d4108 @responses={}, @variant=[]>
```

Any ideas?

**Comment by mjgiarlo** [comment]
@jcoyne This is ready for re-review.

**Pull request merged**